src/Sylius/Bundle/AdminBundle/Resources/views/ProductAttribute/Action/createProductAttribute.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class="ui labeled icon top right floating dropdown button primary link">
+    <i class="plus icon"></i>
+    <span class="text">{{ 'sylius.ui.create'|trans }}</span>
+    <div class="menu">
+        <div class="header">
+            <i class="world icon"></i>
+            {{ 'sylius.ui.type'|trans }}
+        </div>
+        <div class="divider"></div>
+        {% set createRouteName = grid.requestConfiguration.getRouteName('create') %}
+        <a class="item" href="{{ path(createRouteName, { 'type': 'text' }) }}">Text</a>
+        <a class="item" href="{{ path(createRouteName, { 'type': 'textarea' }) }}">Textarea</a>
+        <a class="item" href="{{ path(createRouteName, { 'type': 'checkbox' }) }}">Checkbox</a>
+        <a class="item" href="{{ path(createRouteName, { 'type': 'integer' }) }}">Integer</a>
+        <a class="item" href="{{ path(createRouteName, { 'type': 'percent' }) }}">Percent</a>
```

Comment:
These should be obtained from the container. Please have a look how it is done currently in old admin. We should not hardcode them here. There is a controller action ready and we can render it here. :)